builtin/logical/aws/path_config_root_test.go

```diff
@@ -43,7 +44,8 @@ func TestBackend_PathConfigRoot(t *testing.T) {
 		"identity_token_audience":    "",
 		"identity_token_ttl":         int64(0),
 		"rotation_schedule":          "",
-		"rotation_window":            0,
+		"rotation_period":            time.Duration(0).Seconds(),
```

Comment:
Is there a place we can specify a request (and response comparator) so that this field is "0s" rather than a number?